libretroshare/src/services/autoproxy/rsautoproxymonitor.cc

```diff
@@ -244,20 +256,22 @@ void rsAutoProxyMonitor::taskDone(taskTicket *t, autoProxyStatus::autoProxyStatu
 		t->cb->taskFinished(t);
 		if (t != NULL) {
 			// callack did not clean up properly
-			std::cerr << "(WW) rsAutoProxyMonitor::taskFinish callback did not clean up!" << std::endl;
+			RS_ERR("callback did not clean up!");
+			print_stacktrace();
 			cleanUp = true;
 		}
 	} else if (t->async){
 		// async and no callback
 		// we must take care of deleting
 		cleanUp = true;
 		if(t->data)
-			std::cerr << "(WW) rsAutoProxyMonitor::taskFinish async call with data attached but no callback set!" << std::endl;
+			RS_ERR("sync call with data attached but no callback set!");
```

Comment:
you probably forgot an "a" at the begin of the message

Reply:
nice catch!